web/workflow.R

```diff
@@ -34,7 +34,7 @@ options(error=quote({
 # Open and read in settings file for PEcAn run.
 args <- commandArgs(trailingOnly = TRUE)
 if (is.na(args[1])){
-  settings <- PEcAn.settings::read.settings("pecan.xml")
+  settings <- PEcAn.settings::read.settings("dummy.settings.xml")
```

Comment:
you should definitely NOT change this default in the main workflow (or if you do, you shouldn't commit it!), that will break gobs of stuff.